src/Bootstrap/SetupDatabase.php

```diff
@@ -0,0 +1,66 @@
+<?php
+declare(strict_types=1);
+
+namespace Longman\TelegramBot\Bootstrap;
+
+use Illuminate\Database\Capsule\Manager as Capsule;
+use Illuminate\Database\ConnectionResolver;
+use Illuminate\Events\Dispatcher;
+use Longman\TelegramBot\Application;
+
+use PDO;
+use function array_filter;
+use function env;
+use function extension_loaded;
+
+class SetupDatabase
+{
+    public function bootstrap(Application $app)
+    {
+        $config = $this->getConfig();
+        $capsule = new Capsule($app);
+        $capsule->addConnection($config);
+
+        $capsule->setEventDispatcher(new Dispatcher($app));
+
+        $capsule->setAsGlobal();
+
+        // $capsule->bootEloquent();
+
+        $app->instance('db', $capsule);
+
+        $app->bind('db.connection', function ($app) {
+            return $app['db']->connection();
+        });
+
+        $app->bind('db.resolver', function ($app) {
+            $resolver = new ConnectionResolver(['default' => $app['db.connection']]);
+            $resolver->setDefaultConnection('default');
+            return $resolver;
+        });
+
+        $app->enableExternalMySql($capsule->getConnection()->getPdo());
+    }
+
+    private function getConfig(): array
+    {
+        return [
+            'driver'         => 'mysql',
```

Comment:
Looking forward to allowing this to be set via environment variables too :innocent: 
(Why now allow all config parameters to be set via env vars? Like charset, collation, engine, etc.)

How easy do you think it will be to shift the whole DB class to use illuminate/database?

Reply:
> Looking forward to allowing this to be set via environment variables too

:+1:

> How easy do you think it will be to shift the whole DB class to use illuminate/database?

I think it pretty easy. Let's do that in the second iteration. First, let's make fixes in this integration (think about better .env loading, helper functions, etc.)

Reply:
Agreed :ok_hand: